clippy_lints/src/manual_is_ascii_check.rs

```diff
@@ -23,6 +24,14 @@ declare_clippy_lint! {
     ///     assert!(matches!(b'X', b'A'..=b'Z'));
     ///     assert!(matches!('2', '0'..='9'));
     ///     assert!(matches!('x', 'A'..='Z' | 'a'..='z'));
+    ///
+    ///     assert!((b'0'..=b'9').contains(&b'0'));
+    ///     assert!((b'a'..=b'z').contains(&b'a'));
+    ///     assert!((b'A'..=b'Z').contains(&b'A'));
```

Comment:
Looking at this example, I'd suggest removing the `assert!()` macro around the expression. The code should still compile and make it easier to focus on the expression in question. I'd also say that just the normal letter ranges below are enough. This should mainly give the user an idea what is being linted.